components/tools/OmeroPy/src/omero/clients.py

```diff
@@ -223,6 +223,7 @@ def _initData(self, id):
             # OPENSSL_VERSION_INFO not available for 2.6, fall back to default
             self._optSetProp(id, prop, "HIGH:ADH")
 
+        self._optSetProp(id, "IceSSL.VerifyDepthMax", "0")
```

Comment:
As discussed in slack, I'd go for some limit that users can then turn off. Based on nothing in particular, I'd say 6. (ZeroC went from 2 to 3 for 3.6.0)